bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/instructions.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
Please move this file in a folder named "update"

Reply:
Done